### PR TITLE
feat(datasource,layer): add MVT(MapBox Vector Tile) and Vector Tile l…

### PR DESCRIPTION
**Pull request by drekss**
…ayer supported as new datasource

### CODE REVIEW

**Comment by drekss** [comment]
* feat(datasource,layer): add MVT(MapBox Vector Tile) and Vector Tile layer supported as new datasource.

* ne comporte pas le onLayerStatusChange, car il n'est pas dans la branche alpha.

File: demo/src/contexts/context3.json [hunk 1]
```diff
@@ -16,6 +16,38 @@
...
+        "stroke": ["red"],
+        "label": "geounit"
+      },
+      "sourceOptions": {
+        "type": "mvt",
+        "url": "https://ahocevar.com/geoserver/gwc/service/tms/1.0.0/ne:ne_10m_admin_0_countries@EPSG:900913@pbf/{z}/{x}/{-y}.pbf"
+      }
+    },
+    {
+      "title": "bureaux",
```
Comment (mbarbeau) [inline demo/src/contexts/context3.json]: remove this example since the url is not public

Comment (mbarbeau) [reply demo/src/contexts/context3.json]: also remove png, they are not useful without the example

File: packages/geo/src/lib/layer/shared/layer.service.ts [hunk 2]
```diff
@@ -126,11 +132,39 @@ export class LayerService {
       const source = layerOptions.source as ArcGISRestDataSource;
       style = source.options.params.style;
     }
+    if (layerOptions.styleByAttribute) {
+      const serviceStyle = this.styleService;
+      // tslint:disable-next-line:only-arrow-functions
```
Comment (mbarbeau) [inline packages/geo/src/lib/layer/shared/layer.service.ts]: there is a special reason to disable the lint ?

File: packages/geo/src/lib/layer/shared/layer.service.ts [hunk 3]
```diff
@@ -126,11 +132,39 @@ export class LayerService {
...
   }
+
+  private createVectorTileLayer(layerOptions: VectorTileLayerOptions): VectorTileLayer {
+    let style;
+    if (layerOptions.style !== undefined) {
+      style = this.styleService.createStyle(layerOptions.style);
+    }
+    if (layerOptions.styleByAttribute) {
+      const serviceStyle = this.styleService;
+      // tslint:disable-next-line:only-arrow-functions
```
Comment (mbarbeau) [inline packages/geo/src/lib/layer/shared/layer.service.ts]: there is a special reason to disable the lint ?

**Pull request closed without merging**